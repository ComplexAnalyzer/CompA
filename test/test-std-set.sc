def main()
{
    a = [1,9,6,2,2,3,9];
    d = set_fun(a);
    print("set_fun");
    (0<=i<#d) print(d[i]);
    d = set(a);
    print("set");
    (0<=i<#d) print(d[i]);

}

def set_fun(a) {
    tmp = []; 
    (0<=i<#a |  a[i]?tmp == false) tmp = tmp + [a[i]];
    return tmp;
}
