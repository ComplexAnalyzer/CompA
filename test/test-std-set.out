set_fun
1
9
6
2
3
set
1
9
6
2
3
